### ISSUE TITLE
Ad Comment would be very nice

### ISSUE THREAD

**Issue by poltrian** [mrzapp/nextcloud-cookbook#8]
Hi,
comment like starts or Like, or better you can write something "worst ever" "love" "you could add ... to ... make it better

**Comment by mrzapp** [comment]
@poltrian Yeah, could be neat. Comments are actually supported in the [schema.org standard](https://schema.org/Recipe). We can use the NextCloud user credentials for the user information.

**Comment by Teifun2** [comment]
Where would you add the user credentials to a comment? to `author` or maybe a different schema field?

**Comment by mrzapp** [comment]
@Teifun2 the schema.org standard has a "comment" field we could use

**Comment by Teifun2** [comment]
@mrzapp Yes i did see that. But if we want to store the credentials of the user + its comments, how do we do this? According to schema.org a [Comment](https://schema.org/Comment) can have a Text (which would be the actual comment?) but what field would contain the user credentials? author?

**Comment by mrzapp** [comment]
@Teifun2 exactly, the `author` field is meant for that. Your proposal is exactly how I would do it too.

**Comment by Teifun2** [comment]
Although i think this is a very good feature i would leave it aside for the moment and first focus on #116 #47 and potentially #11 . I think the aspect of storing your own recipes and using them should be the main task. Sharing and collaborative work can come when the individuals database is at a certain level.

**Comment by spikusmax** [comment]
I second this one. One could use the tagging to indicate stars already but it would be nice for that the be represented visually.  I would really like to see dated comments that would enable technical notes. E.g. "needs 60min cooking not 45 in our shitty oven".  Less worried about the collaborative side of comments.

**Comment by Mannshoch** [comment]
May if you Update a Cooking step a popup appear which request to check if comments are out of date and could be removed.

**Comment by christianlupus** [comment]
This is related to #323, #342 and potentially #364.

**Comment by GutHib** [comment]
"needs 60min cooking not 45 in our shitty oven"

This. Or comments like:

"Jan. 21: Used ordinary onion instead of scallion, and butternut pumpkin instead of hokkaido. Pumpkin took a bit longer to soften, but turned out great." Or:

"For some reason, meat turned out tough. Screw the supermarket. Buy at the butcher next time. Also, try less heat."

For me, keeping track of success and failure is an important part of recipe evolution - it's what helps you improve. To me, this isn't related to rating or collaboration - just a simple area for private notes.

Probably, comment field would be best at the bottom (after all, that's where you'd scribble manual notes on a printed recipe).

[Of course, if everyone could arrange order to their liking, that would be even better, but let's not be too greedy now. :-)) ]

**Comment by CDubs00** [comment]
Is this request dead? I came looking for just this feature for thinks like (optional items) like:

### Mix-ins (optional)
- Caramel morsels 
- White/Milk Chocolate chips
- Raisins

**Comment by Mannshoch** [comment]
Meanwhile, I would like a way to publish and find receipts on Fediverse. 
I'm a tiny bit hopeful because [Nextcloud Social](https://github.com/nextcloud/social/issues/1993) seems working again.
